ga4gh/client.py

```diff
@@ -42,21 +41,12 @@ def runRequest(self, request, url, protocolClass, listAttr):
             headers = {"Content-type": "application/json"}
             # make sure we correctly join with/out trailing slashes
             fullUrl = posixpath.join(self._urlPrefix, url)
-            # TODO Can we get requests to output debugging information when
-            # debugLevel > 0?
             authUrl = self._addAuth(fullUrl)
             response = requests.post(
                 authUrl, jsonString, headers=headers, verify=False)
             response.raise_for_status()
             jsonString = response.text
             self._bytesRead += len(jsonString)
-            if self._debugLevel > 1:
```

Comment:
Yes, I think I'd rather just leave this as it is until we do proper logging. To fix this issue, just add a quick print of the same form for the request when `self._debugLevel > 1`.